services/gateway/shared/regex.js

```diff
@@ -18,8 +18,8 @@ const ADDRESS_LISK32 = /^lsk[a-hjkm-z2-9]{38}$/;
 const NONCE = /^[0-9]+$/;
 const TIMESTAMP_RANGE = /^(?:(?:\d+)|(?::(?:\d+))|(?:(?:\d+):(?:\d+)?))$/;
 const HEIGHT_RANGE = /^(?:(?:\d+)|(?::(?:\d+))|(?:(?:\d+):(?:\d+)?))$/;
-const NAME = /^[\w!@$&.]{1,20}$/;
-const NAME_CSV = /^[\w!@$&.,]{1,}$/;
+const NAME = /^[\w!@$&.]{3,20}$/;
```

Comment:
why the regex change?

Reply:
The regex was updated to integrate the restrictions we had on the gateway: https://github.com/LiskHQ/lisk-service/pull/1368/files#diff-0035135774944e209dca330a711431d95be645382b8ccf059ac27ed2535b60adL27-L29